AlertaDengue/dados/dbdata.py

```diff
@@ -3,22 +3,31 @@
  Alertadengue.
 
 """
-from sqlalchemy import create_engine
-from django.core.cache import cache
 from collections import defaultdict
 from datetime import datetime, timedelta
+from typing import Dict, List, Callable
+
+from sqlalchemy import create_engine
+from django.core.cache import cache
+import pandas as pd
+import numpy as np
+import ibis
+from ibis import config as cf
+from ibis.sql.postgres import existing_udf
 
 # local
-from .episem import episem, episem2date
+from dados.episem import episem, episem2date
 from ad_main import settings
 
-import pandas as pd
-import numpy as np
+with cf.config_prefix('sql'):
+    k = 'default_limit'
```

Comment:
remove this intermediate `k` variable
and use directly: `cf.set_option('default_limit', None)`

Reply:
> remove this intermediate `k` variable
> and use directly: `cf.set_option('default_limit', None)`

Done!